integration_tests/poetry.lock

```diff
@@ -107,9 +107,9 @@ optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
```

Comment:
afaik the lock file shouldn't be modified separately by hand -- it should be generated from https://github.com/crypto-org-chain/chain-main/blob/master/integration_tests/pyproject.toml#L22